docs/static/releasenotes.asciidoc

```diff
@@ -119,6 +146,33 @@ The solution is to apply the default change contained in the newer 'jvm.options'
 [[logstash-8-16-1]]
 === Logstash 8.16.1 Release Notes
 
+[[known-issues-8-16-1]]
+==== Known issue
+
+[[known-issue-8-16-1-json_lines]]
+===== "Input buffer full" error with {ls} 8.16.0, 8.16.1, or 8.17.0
```

Comment:
It seems kinda weird to have 8.17.0 listed in the 8.16.1 and 8.16.0 release notes.  
However, this information is complete and correct, and I'm inclined to leave it. LMKWYT.

Note to self @karenzone: Make sure any change here gets propagated to 8.16.1 and 8.16.0 sections.

Reply:
It does, and it's not something we typically do. 

However, the fact that this is not affected in `8.16.2` may make this appropriate to be explicit about which versions are affected to ensure that users are not confused, and don't think we have just missed `8.16.2`.

It's something that could affect us in the future, as we support multiple branches, and the bug story isn't as simple as it used to be, so it's probably up for debate how we want to approach this from a messaging standpoint.